From 03d1bee8cf6ce569d31104750026cf5dbcbed0ac Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Tue, 27 Nov 2018 14:42:48 +0800
Subject: [PATCH 5203/5242] MLK-20469: ov5640: correct parallel ov5640 QVGA
 frame rate setting

commit  4bfc24b2323756d313747ad59f8fd5e3275b58d2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Register setting for ov5640 30fps is the same with 15fps, so
correct it

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit 80b4f71476ffa3f45de01ccbac88fb131a3c444c)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/ov5640_v3.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/imx8/ov5640_v3.c b/drivers/media/platform/imx8/ov5640_v3.c
index 9621d88..03c15f4 100644
--- a/drivers/media/platform/imx8/ov5640_v3.c
+++ b/drivers/media/platform/imx8/ov5640_v3.c
@@ -494,7 +494,7 @@ struct ov5640 {
 	{0x4004, 0x02, 0, 0}, {0x3002, 0x1c, 0, 0}, {0x3006, 0xc3, 0, 0},
 	{0x4713, 0x03, 0, 0}, {0x4407, 0x04, 0, 0}, {0x460b, 0x35, 0, 0},
 	{0x460c, 0x22, 0, 0}, {0x4837, 0x22, 0, 0}, {0x3824, 0x02, 0, 0},
-	{0x5001, 0xa3, 0, 0}, {0x3034, 0x1a, 0, 0}, {0x3035, 0x21, 0, 0},
+	{0x5001, 0xa3, 0, 0}, {0x3034, 0x1a, 0, 0}, {0x3035, 0x11, 0, 0},
 	{0x3036, 0x46, 0, 0}, {0x3037, 0x13, 0, 0},
 };
 
-- 
1.7.9.5

